docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,160 @@
+# Faker connector
+
+The Faker connector generates realistically looking, random data matching a
+defined structure.
+
+## Configuration
+
+The connector can create data in Trino and allowing querying it. Create a
+catalog properties file that specifies the Faker connector by setting the
+`connector.name` to `faker`.
+
+For example, to access data as the `generator` catalog, create the file
+`etc/catalog/generator.properties`. Replace the connection properties as
+appropriate for your setup:
+
+
+```text
+connector.name=faker
+faker.null-probability=0.1
+faker.default-limit=1000
+```
+
+## Examples
```

Comment:
I kind think we should have the examples after the type mapping and sql support .. and yes.. thats not the case in other connector docs.

Reply:
I renamed this section to `Usage`. It's really important to show that the basic usage is only 2 queries, without having to list all columns.